components/Order/interfaces.rst

```diff
@@ -0,0 +1,8 @@
+Interfaces
+==========
+
```

Comment:
All interfaces should be in this file, divided in two categories: `Model Interfaces` and `Service Interfaces`.